tests/testLoadAstrometryNetObjects.py

```diff
@@ -0,0 +1,214 @@
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the LSST License Statement and 
+# the GNU General Public License along with this program.  If not, 
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+import os
+import unittest
+import eups
+
+import lsst.utils.tests as utilsTests
+import lsst.daf.base as dafBase
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.meas.astrom as measAstrom
+
+class TestLoadAstrometryNetObjects(unittest.TestCase):
+
+    def setUp(self):
+        mypath = eups.productDir("meas_astrom")
+        # Set up local astrometry_net_data
+        datapath = os.path.join(mypath, 'tests', 'astrometry_net_data', 'photocal')
+        eupsObj = eups.Eups(root=datapath)
+        ok, version, reason = eupsObj.setup('astrometry_net_data')
+        if not ok:
+            raise ValueError("Need photocal version of astrometry_net_data (from path: %s): %s" %
+                             (datapath, reason))
+        self.datapath = datapath
+        self.config = measAstrom.LoadAstrometryNetObjectsTask.ConfigClass()
+
+        self.bbox = afwGeom.Box2I(afwGeom.Point2I(0, 0), afwGeom.Extent2I(3001, 3001))
+        self.ctrPix = afwGeom.Point2I(1500, 1500)
+        metadata = dafBase.PropertySet()
+        metadata.set("RADECSYS", "FK5")
+        metadata.set("EQUINOX", 2000.0)
+        metadata.set("CTYPE1", "RA---TAN")
+        metadata.set("CTYPE2", "DEC--TAN")
+        metadata.set("CUNIT1", "deg")
+        metadata.set("CUNIT2", "deg")
+        metadata.set("CRVAL1", 215.5)
+        metadata.set("CRVAL2",  53.0)
+        metadata.set("CRPIX1", self.ctrPix[0] + 1)
+        metadata.set("CRPIX2", self.ctrPix[1] + 1)
+        metadata.set("CD1_1",  5.1e-05)
+        metadata.set("CD1_2",  0.0)
+        metadata.set("CD2_2", -5.1e-05)
+        metadata.set("CD2_1",  0.0)
+        self.wcs = afwImage.makeWcs(metadata)
+        self.desNumStars = 270
+
+    def tearDown(self):
+        del self.ctrPix
+        del self.wcs
+        del self.config
+
+    def testLoadObjectsInBBox(self):
+        """Test loadObjectsInBBox
+        """
+        loadANetObj = measAstrom.LoadAstrometryNetObjectsTask(config=self.config)
+
+        loadRes = loadANetObj.loadObjectsInBBox(bbox=self.bbox, wcs=self.wcs, filterName="r")
+        refCat = loadRes.refCat
+        self.assertEqual(loadRes.fluxField, "r_flux")
+        self.assertEqual(len(refCat), self.desNumStars)
+        self.assertObjInBBox(refCat=refCat, bbox=self.bbox, wcs=self.wcs)
+        schema = refCat.getSchema()
+        filterNameList = ['r'] # ['u', 'g', 'r', 'i', 'z']
```

Comment:
I respectfully disagree. Your suggestion certainly uses fewer characters but I feel it obscures what is going on and forces single-letter filter names.